core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -77,8 +83,9 @@ class ConsumerBounceTest extends AbstractConsumerTest with Logging {
     }
   }
 
-  @Test
-  def testConsumptionWithBrokerFailures(): Unit = consumeWithBrokerFailures(10)
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
+  @ValueSource(strings = Array("zk", "kraft", "kraft+kip848"))
+  def testConsumptionWithBrokerFailures(quorum: String): Unit = consumeWithBrokerFailures(10)
```

Comment:
This test is very flaky with kraft and kraft+kip848

Reply:
Any idea why? Is it only this one or others are well?

Reply:
This is the only one flaky. 

The test fails because somehow the consumer coordinator loses its generation. I guess it's related to timeout issue, because zk broker also fails when setting break points. Maybe something also needs time to propagate and reaches timeout before consumer coordinator updates the generation.